google/cloud/bigquery/table.py

```diff
@@ -1723,7 +1766,7 @@ def to_arrow_iterable(
 
         bqstorage_download = functools.partial(
             _pandas_helpers.download_arrow_bqstorage,
-            self._project,
+            self._bqstorage_project,
```

Comment:
Looks like we now have two possible project values, does this mean a `RowIterator` can correspond to different projects for storage APIs compared to other APIs?